client/src/App.js

```diff
@@ -1,12 +1,14 @@
 import { Route, Routes } from "react-router-dom";
-
+import 'bootstrap/dist/css/bootstrap.min.css';
 import About from "./pages/About";
 import Home from "./pages/Home";
+import TeacherForm from "./pages/TeacherForm";
 
 const App = () => (
 	<Routes>
 		<Route path="/" element={<Home />} />
 		<Route path="/about/this/site" element={<About />} />
+		<Route path="/registration" element={<TeacherForm />} />
```

Comment:
maybe we can make it so that teacher registration is behind an admin route so something like /admin/teacher-registration

Reply:
All done